docs/cluster.md

```diff
@@ -0,0 +1,396 @@
+m_nfw_200 = 10.0 # in code_mass
+```
+which adds a gravitational acceleration defined by
+
+$$
+g_{\text{NFW}}(r) = 
+  \frac{G}{r^2}
+  \frac{M_{NFW}  \left [ \ln{\left(1 + \frac{r}{R_{NFW}} \right )} - \frac{r}{r+R_{NFW}} \right ]}
+        { \ln{\left(1 + c_{NFW}\right)} - \frac{ c_{NFW}}{1 + c_{NFW}} }
+$$
+The scale radius $R_{NFW}$ for the NFW profile is computed from
+$$
+R_{NFW} = \left ( \frac{M_{NFW}}{ 4 \pi \rho_{NFW} \left [ \ln{\left ( 1 + c_{NFW} \right )} - c_{NFW}/\left(1 + c_{NFW} \right ) \right ] }\right )^{1/3}
+$$
+where the scale density $\rho_{NFW}$ is computed from 
+$$
+\rho_{NFW} = \frac{200}{3} \rho_{crit} \frac{c_{NFW}^3}{\ln{\left ( 1 + c_{NFW} \right )} - c_{NFW}/\left(1 + c_{NFW} \right )}.
+$$
+The critical density $\rho_{crit}$ is computed from
+$$
+    \frac{3 H_0^2}{8 \pi G}.
+$$
+
+Parameters for both the HERNQUIST and MATHEWS BCG are controlled via:
+```
+<problem/cluster/gravity>
+
+m_bcg_s = 0.001 # in code_mass
+r_bcg_s = 0.004 # in code_length
+```
+where a HERNQUIST profile adds a gravitational acceleration defined by
+
+$$
+ g_{BCG}(r) = G \frac{ M_{BCG} }{R_{BCG}^2} \frac{1}{\left( 1 + \frac{r}{R_{BCG}}\right)^2}
+$$
+
+and a MATHEWS profile adds a gravitational acceleration defined by
+
+$$
+g_{BCG}(r) = \frac{1}{R_{BCG}^2} 
+\left (
+\left [ \frac{\left ( r/R_{BCG}\right )^{ 0.5975}} {3.206 \times 10^{-7}} \right ]^{s_{BCG}} +
+\left [ \frac{\left ( r/R_{BCG}\right )^{ 1.849 }} {1.861 \times 10^{-6}} \right ]^{s_{BCG}}
+\right )^{-1/s_{BCG}}
+$$
+
+where $s_{BCG} = 0.9$ 
+
+Gravitational acceleration from the SMBH is inserted as a point source defined solely by its mass
+```
+<problem/cluster/gravity>
+m_smbh = 1.0e-06 # in code_mass
+```
+
+Some acceleration profiles may be ill-defined at the origin. For this, we provide a smoothing length parameter,
+```
+<problem/cluster/gravity>
+g_smoothing_radius = 0.0 # in code_length
+```
+which works as a minimum r when the gravitational potential is applied. It
+effectively modifies the gravitation acceleration to work as
+
+$$
+\tilde{g} (r) = g( max( r, r_{smooth}))
+$$
+
+By default, the gravitational profile used to create the initial conditions is
+also used as an accelerating source term during evolution. This source term can
+be turned off, letting this gravitational profile to only apply to
+initialization, with the following parameter in `<problem/cluster/gravity>`.
+```
+<problem/cluster/gravity>
+gravity_srcterm = False
+```
+
+## Entropy Profile
+
+The `cluster` problem generator initializes a galaxy-cluster-like system with an entropy profile following the ACCEPT profile
+
+$$
+ K(r) = K_{0} + K_{100} \left ( r/ 100 \text{ kpc} \right )^{\alpha_K}
+ $$
+ 
+where we are using the entropy $K$ is defined as 
+
+$$
+ K \equiv \frac{ k_bT}{n_e^{2/3} }
+ $$
+ 
+This profile is determined by these parameters
+```
+<problem/cluster/entropy_profile>
+k_0 = 8.851337676479303e-121 # in FIXME
+k_100 = 1.3277006514718954e-119 # in FIXME
```

Comment:
I need to add the units of these in code units for the FIXME. You're correct that we need `Real` to be `float64` here

Reply:
I'm not sure how to bring this quantity closer to unity, perhaps divided by mean particle mass. This could be explored in a cleanup PR later.

For now, the correct units have been added to the documentation